engine/dlib/src/remotery/update_remotery.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Followind our upgrade guidelines in readme.md, this script makes it easy to update to the latest version of Remotery